components/sst_importer/src/sst_importer.rs

```diff
@@ -377,10 +378,8 @@ impl SstImporter {
 
             INPORTER_APPLY_COUNT.with_label_values(&["key_meet"]).inc();
             let ts = Key::decode_ts_from(event_iter.key())?;
-            if ts > TimeStamp::new(restore_ts) {
-                // we assume the keys in file are sorted by ts.
-                // so if we met the key not satisfy the ts.
-                // we can easily filter the remain keys.
+            if ts > TimeStamp::new(restore_ts) || ts < TimeStamp::new(start_ts) {
```

Comment:
I'm afraid filter ts less than start_ts is not right. cause we may want to restore some event in default earlier.

Let add it in next version.

Reply:
The start_ts is 
'startTS' when file.cf == "write", 
'shiftTS' when file.cf ==  "default".
https://github.com/pingcap/tidb/pull/34703/files#r873913758